src/main/java/com/hermes/recruitment/model/Candidat.java

```diff
@@ -0,0 +1,49 @@
+package com.hermes.recruitment.model;
+
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "candidat")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Candidat implements Serializable {
+    @Id
+    @GeneratedValue
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @Column(name = "surname", nullable = false)
+    private String surname;
+
+    @Column(name = "firstname", nullable = false)
+    private String firstname;
+
+    @Column(name = "mail", nullable = false)
+    private String mail;
+
+    @Column(name = "phone", nullable = false, length = 10)
+    private String phone;
+
+    @Column(name = "department", nullable = false)
+    private String department;
+
+    @Column(name = "motivation", nullable = false)
+    private String motivation;
+
+    public Candidat(String surname, String firstname, String mail, String phone, String department, String motivation)
+    {

```

Comment:
Inline brace with method signature
This coding style is specific to C#, not Java